.nix/config.nix

```diff
@@ -15,6 +16,7 @@
       mathcomp-single-planB-src.job = true;
       mathcomp-single-planB.job = true;
     } // common;
+    "coq-8.13".ocamlPackages.elpi.override.version = "v1.13.6";
```

Comment:
`elpi` is "manually" overriden in the `coq-elpi` derivation, do the override in `.nix/coq-overlays/coq-elpi/default.nix`